cluster/cluster.go

```diff
@@ -267,6 +267,16 @@ type Cluster interface {
 	ClusterAlerts
 }
 
+//Secrets generic secret store operations
+type Secrets interface {
```

Comment:
should I create separate package for each of PX API sections(service, diags command) which will contain interface and there implementation, or should all those clubbed together

Reply:
I think it is ok to have the interface here.